src/protocol/common.rs

```diff
@@ -20,18 +20,12 @@ pub enum Value {
     Null,
     /// A boolean value.
     Bool(bool),
-    /// An unsigned int 32.
-    U32(u32),
-    /// A signed int 32.
-    I32(i32),
     /// An unsigned int 64.
-    U64(u64),
+    UnsignedInteger(u64),
```

Comment:
Just leave these names, ideally, or choose a shorter version (`Int`, `UInt`, and `Float`) if need be.